automationtestapp/src/main/java/com/microsoft/aad/automation/testapp/MainActivity.java

```diff
@@ -128,6 +145,12 @@ private void launchAuthenticationInfoActivity(int flowCode) {
         this.startActivity(intent);
     }
 
+    private void openWevViewActivity(){
+        final Intent intent = new Intent();
+        intent.setClass(mContext, WebViewActivity.class);
```

Comment:
Nit : we can use Intent's constructor to set class and context directly like

 `final Intent intent = new Intent(mContext, WebViewActivity.class);`

Reply:
I'll fix since I'm fixing the typo